packages/mdc-rtl/README.md

```diff
@@ -28,76 +26,20 @@ npm install --save @material/rtl
 
 ## Usage
 
-Simply `@import "@material/rtl/mixins";` and start using the mixins. Each mixin is described below.
+### Sass Mixins
 
-### mdc-rtl
+`$base-property` is a base box-model property. e.g. margin / border / padding.
+`$pos` is a horizontal position property, either "left" or "right".
 
-```scss
-@mixin mdc-rtl($root-selector: null)
-```
-
-Creates a rule that will be applied when an MDC-Web component is within the context of an RTL layout.
-
-Usage Example:
-
-```scss
-.mdc-foo {
-  position: absolute;
-  left: 0;
 
-  @include mdc-rtl {
-    left: auto;
-    right: 0;
-  }
+| Mixin | Description |
+| ----------------------------------------------- | - |
+| `mdc-rtl($root-selector)` | Creates a rule that is applied when the root element is within an RTL context |
+| `mdc-rtl-reflexive-box($base-property, $default-direction, $value, $root-selector)` | Applies the value to the `#{$base-property}-#{$default-direction}` property  in a LTR context, and flips the direction in an RTL context. This mixin zeros out the original value in an RTL context. |
+| `mdc-rtl-reflexive-property($base-property, $left-value, $right-value, $root-selector)` | Emits rules that assign `#{$base-property}`-left to `#{left-value}` and `#{base-property}`-right to `#{right-value}` in a LTR context, and vice versa in a RTL context. |
+| `mdc-rtl-reflexive-position($pos, $value, $root-selector)` | Applies the value to the specified position in a LTR context, and flips the direction in an RTL context. |
 
```

Comment:
I want developers to read about what each mixin accomplishes. This lets developers decide where they can use the mixins in their own code. Once a developer is using a mixin, we can upgrade the implementation of the mixin **so long as it still accomplishes the same original goal**. 

When we document code examples of the actual CSS output of the mixin, we risk developers using a mixin simply because they want that exact output...NOT because they want to accomplish the same goal the mixin aims to solve.

So we can document examples of _where_ to use this mixins, but we should avoid documenting the actual CSS output. The CSS output is an implementation detail.

...I looked at our code and I couldn't actually find great examples of us using these properties...Theres one in checkbox, but it's more confusing to me than helpful.